docker/modelzooserver/Dockerfile

```diff
@@ -11,14 +11,14 @@ RUN /bin/sh -c 'if [ "$FIND_FASTED_MIRROR" == "true" ]; then source find_fastest
     echo "Choose the fastest Alpine source ..." && \
     choose_fastest_alpine_source && \
     echo "Choose the fastest PIP source ..." && \
-    choose_fastest_pip_source && \
+    choose_fastest_pip_source; fi && \
```

Comment:
I believe this code fixes a bug, but not installing the bash. Maybe, we should change the PR title.

Reply:
We have `apk add --no-cache python3 py3-pip sudo bash` after `fi` here. This statement will install bash.